src/install/config/python.yaml

```diff
@@ -0,0 +1,750 @@
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-starlette-web-framework
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-starlette-web-framework
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-tornado-6-web-framework
+
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado-3
+  - /node/1971
+  - /docs/python/python-agent-and-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/nr-only-python-agent-tornado
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-tornado-experimental
+  - /docs/agents/python-agent/hosting-mechanisms/introductory-tornado-4-support
+  - /docs/agents/python-agent/web-frameworks-servers/using-trace-decorators-tornado-coroutines
+  - /docs/agents/python-agent/trace-decorators-tornado-coroutines
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-tornado-6-web-framework
+
+  - /docs/agents/python-agent/web-frameworks-servers/uvicorn
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-uvicorn-web-server
+  - /docs/agents/python-agent/web-frameworks-servers/uvicorn-web-server
+  - /docs/apm/agents/python-agent/web-frameworks-servers/uvicorn
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-uwsgi-web-server
+  - /docs/python/python-agent-and-uwsgi
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-uwsgi
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-uwsgi
+  - /docs/apm/agents/python-agent/web-frameworks-servers
+  - /docs/agents/python-agent/web-frameworks-and-servers
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-uwsgi-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-waitress-web-server
+  - /docs/python/python-agent-and-waitress
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-and-waitress
+  - /docs/agents/python-agent/hosting-mechanisms/python-agent-waitress
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-waitress-web-server
+
+  - /docs/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
+  - /docs/python/python-agent-and-web2py
+  - /docs/agents/python-agent/web-frameworks/python-agent-and-web2py
+  - /docs/agents/python-agent/web-frameworks/python-agent-web2py
+  - /docs/apm/agents/python-agent/web-frameworks-servers/python-agent-web2py-web-framework
+
+
+# -----------------------------------------------------------------
+# Define the categories for each type of question you'll be asking:
+# -----------------------------------------------------------------
+appInfo:
+
+
+
+  - optionType: python-non-web
+    label: "We'll show you how to instrument your non-web Python app"
+    placeholder: 'Make a selection:'
```

Comment:
I don't see the value in having these dropdown menus collapsed by default. That seems odd to me and leads to a lot of extra, unnecessary clicking in my view.

Reply:
The way it shows "Make a selection" but then you have to click to see the "yes"/"no" buttons.

Reply:
@paperclypse how would we force the question options to be displayed by default?